docs/pip/environments.md

```diff
@@ -59,12 +59,14 @@ Since uv has no dependency on Python, it can install into virtual environments o
 For example, setting `VIRTUAL_ENV=/path/to/venv` will cause uv to install into `/path/to/venv`,
 regardless of where uv is installed. Note that if `VIRTUAL_ENV` is set to a directory that is
 **not** a [PEP 405 compliant](https://peps.python.org/pep-0405/#specification) virtual environment,
-it will be ignored.
-
-uv can also install into arbitrary, even non-virtual environments, with the `--python` argument
-provided to `uv pip sync` or `uv pip install`. For example,
-`uv pip install --python /path/to/python` will install into the environment linked to the
-`/path/to/python` interpreter.
+it will be ignored. The path to the virtual environment can also be provided with the `--python`
+option, e.g., `uv pip install --python /path/to/venv ...`.
+
+uv can also install into arbitrary, even non-virtual environments, with the `--python` option. For
+example, `uv pip install --python /path/to/python` will install into the environment linked to the
+`/path/to/python` interpreter regardless of whether or not it is a virtual environment. As with
+virtual environments, the provided path can be the root directory of an environment instead of the
```

Comment:
Maybe "or the Python executable itself"?

Reply:
Well we just said this before. I think this needs to be restructured further. Will address.